invenio_drafts_resources/services/records/components/base.py

```diff
@@ -226,6 +226,19 @@ def _publish_edit(self, identity, draft, record):
         # Teardown the bucket and files created in edit().
         self._purge_bucket_and_ovs(draft_files)
 
+    def _check_file_completed(self, file_record):
+        """Check if file upload has completed."""
+        # Check if RDMDraftFile file has OV assigned
+        # - if not, the upload is ongoing or has failed (fail handled elsewhere)
+        # prevents ambiguous errors when trying to publish a record with
+        # ongoing upload (cannot get storage class of None if OV is not set)
+        has_attached_object = file_record.file is not None
+        if not has_attached_object:
+            return
```

Comment:
```suggestion
            return False
```
minor: I like to be explicit about return values. None will also evaluate to `False` so feel free to ignore